code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm

```diff
@@ -521,7 +521,22 @@
 				H << "<span class='danger'>Your stomach hurts too much as pressure builds up inside of you.</span>"
 				H.adjustToxLoss(6*REM)
 	..()
-
+  
+/datum/chemical_reaction/fartium_explosion
```

Comment:
I can set a DNM until the admin logging @TheOneAndOnlyCreeperJoe 

Also, yeah, was late night/early morning code, will work on it when I return